meteor/server/api/blueprints/context/adlibActions.ts

```diff
@@ -108,19 +112,48 @@ export class ActionExecutionContext extends ShowStyleContext implements IActionE
 	public takeAfterExecute: boolean
 
 	constructor(
+		contextInfo: UserContextInfo,
 		cache: CacheForRundownPlaylist,
-		notesContext: NotesContext,
 		studio: Studio,
 		rundownPlaylist: RundownPlaylist,
 		rundown: Rundown
 	) {
-		super(studio, cache, rundown, rundown.showStyleBaseId, rundown.showStyleVariantId, notesContext)
+		super(contextInfo, studio, cache, rundown, rundown.showStyleBaseId, rundown.showStyleVariantId)
 		this._cache = cache
 		this.rundownPlaylist = rundownPlaylist
 		this.rundown = rundown
 		this.takeAfterExecute = false
 	}
 
+	notifyUserError(message: string, params?: { [key: string]: any }, trackingId?: string): void {
+		if (this.blackHoleNotes) {
```

Comment:
Am I missing something, or will this.blackHoleNotes always be false? It's readonly and doesn't get a value in the constructor?

Reply:
I missed assigning it a value in the constructor here..